libs/sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -1095,6 +1097,9 @@ PERCENT: '%';
 CONCAT: '||';
 CAST_OPERATOR: '::';
 SEMICOLON: ';';
+BITWISE_AND: '&';
+BITWISE_OR: '|';
+BITWISE_XOR: '^' | '#';
```

Comment:
Why do we want to support `^` for XOR? At PostgreSQL the `^` is used for exponentiation, such a pg-user would not expected that this used for XOR.  If we have no concrete use-case I'd suggest to not support `^` for XOR.

Reply:
I was going to align with "classic" operator (for exp we have pow) but I haven't considered PG users, good point. Pushed a fixup in https://github.com/crate/crate/pull/13278/commits/8c2227bf894319751cf4726ef7d7579690efe703